assets/js/app.js

```diff
@@ -1,92 +1,146 @@
+      el.parentElement.remove();
+    }
+    localStorage.setItem('books', JSON.stringify(books));
+  }
 
-  localStorage.setItem('books', JSON.stringify(books));
-};
+  static clearFields() {
+    document.querySelector('#bookTitle').value = '';
+    document.querySelector('#bookAuthor').value = '';
+  }
+}
 
 // Event: Display Books
-document.addEventListener('DOMContentLoaded', displayBooks);
+document.addEventListener('DOMContentLoaded', BookUserInterface.displayBooks);
 
 // Event: Add a Book
 document.querySelector('#book-form').addEventListener('submit', (e) => {
   // Prevent actual submit
   e.preventDefault();
+
   // Get form values
   const bookTitle = document.querySelector('#bookTitle').value;
   const bookAuthor = document.querySelector('#bookAuthor').value;
-  // Create unique id for each book
-  const index = Date.now();
 
   // Instatiate book
-  const book = new Book(index, bookTitle, bookAuthor);
+  const book = new Book(bookTitle, bookAuthor);
 
   // Add Book to BookUserInterface
-  addBook(book);
+  BookUserInterface.addBookToList(book);
+
+  // Add book to SavedBooks
+  SavedBooks.addBook(book);
+
+  // Clear fields
+  BookUserInterface.clearFields();
 });
 
 // Event: Remove a Book
-listBooks.addEventListener('click', (e) => {
-  // Get the position of DOM parent element of the button clicked
-  const position = Array.from(e.target.parentNode.parentNode.children).indexOf(
-    e.target.parentNode,
+document.querySelector('#book-list').addEventListener('click', (e) => {
+  // Remove book from BookUserInterface
+  BookUserInterface.removeBook(e.target);
+
+  // Remove book from SavedBooks
+  SavedBooks.removeBook(
+    e.target.previousElementSibling.previousElementSibling.textContent,
   );
+});
+
+const currentDate = new Date().toLocaleString();
+document.getElementById('current-date').innerHTML = currentDate;
+
+// display the books list when click the button "List"
+const bookList = document.querySelector('.book-list-container');
+const listBtn = document.querySelector('.listBtn');
+const formContainer = document.querySelector('.form-container');
+
+listBtn.addEventListener('click', () => {
+  bookList.style.display = 'block';
+  formContainer.style.display = 'none';
+});
+
+window.addEventListener('load', () => {
+  bookList.style.display = 'block';
+  formContainer.style.display = 'none';
+});
+
+// display the Add book form  when click the button "Add new"
+const addNewBtn = document.querySelector('.add-new-btn');
+
+addNewBtn.addEventListener('click', () => {
+  bookList.style.display = 'none';
+  formContainer.style.display = 'block';
+});
+
+// display the  Contact section when click the button "Contact"
+const contactBtn = document.querySelector('.contact');
+const contactInfo = document.querySelector('.contact-info');
 
-  removeBook(position);
+contactBtn.addEventListener('click', () => {
+  bookList.style.display = 'none';
+  formContainer.style.display = 'none';
+  contactInfo.style.display = 'block';
```

Comment:
- [ ] I notice that you have a lot of code that you're not using. At the moment these lines don't serve any kind of purpose, please remove them in order to keep your codebase clean and readable. 🤓 🛠️